ModelBundle/Migrations/MongoDB/Version20150722104732.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Migrations\MongoDB;
+
+use AntiMattr\MongoDB\Migrations\AbstractMigration;
+use Doctrine\MongoDB\Database;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20150722104732 extends AbstractMigration
+{
+    /**
+     * @return string
+     */
+    public function getDescription()
+    {
+        return "";
+    }
+
+    public function up(Database $db)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+
+    }
+
+    public function down(Database $db)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+
+    }
+    public function postUp(Database $db)
+    {
+        $result = $this->executeScript($db, 'duplicate_node.js');
```

Comment:
I think that the js file should also have a version number, so you will be able to rollback from the new version to the old one (for exemple, if you rollback the code)